CLI/test.py

```diff
@@ -0,0 +1,15 @@
+from django.test import TestCase
+
+from pyworkflow.pyworkflow import Workflow
+
+
+class WorkflowTest(TestCase):
+
+    def set_up(self):
+        self.workflow = Workflow()
+
+    def test_workflow_init(self):
+        self.assertEqual(self.workflow.name, "a-name")
```

Comment:
The default Workflow name was changed to "Untitled". Might be better to pass in a name argument when calling `Workflow()` and assert based on that value.

Reply:
Please see above, I unintentionally pushed this files. Will get these lines removed.